x/avs/types/expected_keepers.go

```diff
@@ -37,6 +37,7 @@ type OperatorKeeper interface {
 	IsOperator(ctx sdk.Context, addr sdk.AccAddress) bool
 	OptIn(ctx sdk.Context, operatorAddress sdk.AccAddress, avsAddr string) error
 	OptOut(ctx sdk.Context, operatorAddress sdk.AccAddress, avsAddr string) (err error)
+	GetOptedInOperatorListByAVS(ctx sdk.Context, avsAddr string) ([]string, error)
```

Comment:
Will you implement this?

Reply:
Already implemented, on line 197 of the operator.go file